src/CMakeLists.txt

```diff
@@ -80,9 +58,19 @@ endif()
 get_target_property(ESPP_CFLAGS _espressopp COMPILE_OPTIONS)
 message(STATUS "_espressopp compiler flags are: ${ESPP_CFLAGS}")
 
-target_link_libraries(_espressopp ${Boost_LIBRARIES} ${PYTHON_LIBRARIES} MPI::MPI_CXX ${FFTW3_LIBRARIES} ${VAMPIRTRACE_LIBRARIES})
+target_include_directories(_espressopp PUBLIC $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/src> $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/src/include>
+     $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/src>)
```

Comment:
Do we also need `INSTALL_INTERFACE` for the installed version to work?

Reply:
We don't install any headers, so there is no install interface ;-)

Reply:
Ok, so we do not use the raw usage of `_espresso.so` without the Python layer? @stuehn mentioned it once. We do not have raw C++ users?

Reply:
We can always add an install interface once we have users for it.